src/components/Share/__snapshots__/Share.test.js.snap

```diff
@@ -173,6 +173,37 @@ exports[`Share should match snapshot. 1`] = `
       </svg>
     </a>
   </div>
+  <div
+    className="ta-manual-icon"
+  >
+    <a
+      href="null/static/app_trafimage/docs/undefined/Quickstart.pdf"
```

Comment:
can you make a real url appears? wihtout null and undefined values

Reply:
Makes sense, I pass the correct props to the snapshot test.